### PR TITLE
7134 - Custom button adjustments in mobile

### PR DESCRIPTION
**Pull request by yohannahbautista**
**Explain the _details_ for making this change. What existing problem does the pull request solve?**

Additional adjustments in custom button for mobile screen width.

**Related github/jira issue (required)**:

Addresses comment: https://github.com/infor-design/enterprise/issues/7134#issuecomment-1409795239

**Steps necessary to review your pull request (required)**:

- Pull this branch, build and run the app
- Change screen width to mobile
- Go to: http://localhost:4000/components/searchfield/example-buttons.html?theme=classic&mode=light
- Click on searchfield
- Type on searchfield
- Check that icons are aligned

**Included in this Pull Request**:
- [ ] A note to the change log.

### CODE REVIEW

File: docs/CHANGELOG.md [hunk 1]
```diff
@@ -7,9 +7,12 @@
 - `[General]` Project now uses node 18 (18.13.0) for development. All dependencies are updated. ([#6634](https://github.com/infor-design/enterprise/issues/6634))
 - `[General]` Updated to d3.v7 which impacts all charts. ([#6634](https://github.com/infor-design/enterprise/issues/6634))
 
-## v4.81.0 Fixes
+## v4.82.0 Fixes
```
Comment (tmcconechy) [inline docs/CHANGELOG.md:10]: 4.81.0 is correct for the next version..

**Review by tmcconechy — CHANGES_REQUESTED** [review]
The only issue(s) i see are..

- resize to mobile
- click the search to expand
- press the collapse button -> it collapses and then expands again
- click the search to expand
- click out of the field to collapse -> what as it collapses centered then to the side (looks clunky if that can be fixed)
- also the "toolbar" text is extremely small. can you check it?

**Comment by tmcconechy** [comment]
- change log conflict
- on http://localhost:4000/components/searchfield/example-buttons.html?theme=classic&mode=light i still see:
  - the icon is now cut off
  - the animation looks very clunky when collapsing

**Comment by tmcconechy** [comment]
Still seeing a weird animation. Do you think its fixable @yohannahbautista ?
https://watch.screencastify.com/v/llQRSQqlfa0ex1Jr1QsN

**Comment by yohannahbautista** [comment]
> Still seeing a weird animation. Do you think its fixable @yohannahbautista ? https://watch.screencastify.com/v/llQRSQqlfa0ex1Jr1QsN

@tmcconechy  @ericangeles 
Hello, the jittery animation is something that also appears in older versions: https://4660-enterprise.demo.design.infor.com/components/searchfield/example-buttons
It might have something to do with the transition when the width changes. I didn't see any changes in styling midway through it collapsing, it should change to width 34px once the focus and active classes are removed. It might take time to find a solution for it.

**Comment by ericangeles** [comment]
@tmcconechy maybe we can separate this issue?

**Comment by tmcconechy** [comment]
Sure if you need to ~ but it looks pretty clunky to me.

**Review by tmcconechy — APPROVED** [review]

**Review by tmcconechy — APPROVED** [review]

**Review by tmcconechy — APPROVED** [review]

**Review by ericangeles — APPROVED** [review]

**Pull request merged**